projects/apple/tomviz.bundle.cmake

```diff
@@ -32,7 +27,7 @@ install(CODE "
          USE_SOURCE_PERMISSIONS TYPE DIRECTORY FILES
          \"${install_location}/lib/python2.7/site-packages/\")
 
-    if(${itk_condition_variable})
+    if(${itk_ENABLED})
```

Comment:
Last question - why the ${}, I am used to writing this as if(itk_ENABLED) as you had in your extra thing you are deleting above. That always seemed more reliable anyway, otherwise putting quotes around this but that wasn't great either.